CHANGELOG.md

```diff
@@ -38,6 +38,14 @@ Bottom level categories:
 - Hal
 -->
 
+By @bradwerth in [#4203](https://github.com/gfx-rs/wgpu/pull/4023)
+
+### Bug Fixes
+
+#### Metal
+
+- Ensure that MTLCommandEncoder calls endEncoding before it is deallocated.
+
```

Comment:
nitpick(blocking): Whoa, this `CHANGELOG` entry is unexpectedly formatted. Right now, you have outside of the `Unreleased` section:

```md
$credits

$section_headers

- $desc
```

Looks like you need to do at least:

1. Merge with the already existing `Bug Fixes` section below, inside of `Unreleased`.
2. Rearrange your content to be:

  ```
  $section_headers

  - $desc $credits 
  ```

@teoxoy's model has a concrete example of this below, if it helps.